otelcollector/fluent-bit/fluent-bit-daemonset.conf

```diff
@@ -130,6 +130,42 @@
     Skip_Long_Lines On
     Ignore_Older 2m
 
+[INPUT]
+    Name          proc
+    Proc_Name     otelcollector
+    Interval_Sec  10
+    Tag           proc.otelcollector.memvmrss.scrape
+
+[INPUT]
+    Name          proc
+    Proc_Name     metricsextension
+    Interval_Sec  10
+    Tag           proc.metricsextension.memvmrss.scrape
+
+[INPUT]
+    name prometheus_scrape
+    host localhost
+    port 8888
+    tag          promscrape.prometheus.8888
+    scrape_interval 10s
+
+[INPUT]
+    name prometheus_scrape
+    host localhost
+    port 9090
+    tag           promscrape.prometheus.9090
+    scrape_interval 10s
+
+[INPUT]
+    Name cpu
+    Tag  cpu.otelcollector
+    PID  ${OTEL_PID}
+
+[INPUT]
+    Name cpu
+    Tag  cpu.me
+    PID  ${ME_PID}
+
```

Comment:
this seems to be missing the input plugin for targetallocator - 
[[inputs.prometheus]]
  interval = "5m"
  urls = ["http://ama-metrics-operator-targets.kube-system.svc.cluster.local/metrics"]
  fieldpass = ["opentelemetry_allocator_targets","opentelemetry_allocator_collectors_discovered"]
  metric_version = 2
  url_tag = "scrapeUrl"
  response_timeout = "15s"
  name_override = "target_allocator"

Could you pls add that?

Reply:
Thanks Rashmi! yes I will add this.

Reply:
I have added this change and have uploaded the screenshot of flowing telemetry(last screenshot).